src/Components/Admin/Positions/Form/index.js

```diff
@@ -95,110 +71,127 @@ function Form() {
       });
   }, []);
 
-  const onSubmit = (event) => {
-    event.preventDefault();
+  const onSubmit = (formValues) => {
     const positionId = query.get('_id');
 
     if (positionId) {
-      dispatch(
-        updatePosition(positionId, {
-          client: clientIdValue,
-          jobDescription: jobDescriptionValue,
-          vacancy: vacancyValue,
-          professionalProfile: professionalProfileIdValue,
-          isOpen: isOpenValue
-        })
-      ).then((response) => {
+      dispatch(updatePosition(positionId, formValues)).then((response) => {
         if (response) {
           history.push('/admin/positions/list');
         }
       });
     } else {
-      dispatch(
-        createPosition({
-          client: clientIdValue,
-          jobDescription: jobDescriptionValue,
-          vacancy: vacancyValue,
-          professionalProfile: professionalProfileIdValue,
-          isOpen: isOpenValue
-        })
-      ).then((response) => {
+      dispatch(createPosition(formValues)).then((response) => {
         if (response) {
           history.push('/admin/positions/list');
         }
       });
     }
   };
 
+  const validate = (formValues) => {
```

Comment:
You should add a validation in the vacancy field so that it supports only integers